pkg/cloud/aws/services/ec2/instances.go

```diff
@@ -230,11 +242,14 @@ func (s *Service) createInstance(machine *actuators.MachineScope, bootstrapToken
 				),
 				kubeadm.WithLocalAPIEndpointAndPort(localIPV4Lookup, apiServerBindPort),
 			)
+
 			joinConfigurationYAML, err := kubeadm.ConfigurationToYAML(&machine.MachineConfig.KubeadmConfiguration.Join)
 			if err != nil {
 				return nil, err
 			}
 
+			s.scope.Logger.V(3).Info("control plane no 0", "user data", joinConfigurationYAML)
```

Comment:
Left over from debugging?

Reply:
reasonable

Reply:
> Left over from debugging?

yep, I think that this log is useful to help the user to debug how to create the ec2 instance.